src/Controller/Api/v1/DepartmentController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace App\Controller\Api\v1;
+
+use App\Entity\Department;
+use App\Services\DepartmentService;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+#[Route('/api/v1/department', name: 'app_api_v1_department_')]
+class DepartmentController extends AbstractController
+{
+    #[Route('/list', name: 'list', methods: ['GET'])]
+    public function index(EntityManagerInterface $entityManager): JsonResponse
+    {
+        $departments = $entityManager->getRepository(Department::class)->findAll();
+        $data = [];
+        foreach ($departments as $department) {
+            $data[] = [
+                'id' => $department->getId(),
+                'title' => $department->getTitle(),
+                'slug' => $department->getSlug(),
+                'description' => $department->getDescription(),
+            ];
+        }
+
```

Comment:
Can we move this section to any service?

Reply:
Done